File: app/src/main/java/com/vertice/teepop/leaveapp/util/template/DateConverter.java
package com.vertice.teepop.leaveapp.util.template;

import android.databinding.BindingConversion;

import java.text.SimpleDateFormat;
import java.util.Date;

/**
 * Created by VerDev06 on 1/4/2018.
 */

public class DateConverter {

    @BindingConversion
    public static String convertDateToDisplayTime(Date date) {
        return new SimpleDateFormat("yyyy/MM/dd").format(date);
    }
}

File: app/src/main/java/com/vertice/teepop/leaveapp/presentation/DateConverter.kt
package com.vertice.teepop.leaveapp.presentation

import android.annotation.SuppressLint
import androidx.databinding.BindingConversion
import java.text.SimpleDateFormat
import java.util.*

/**
 * Created by VerDev06 on 1/9/2018.
 */
object DateConverter {

    @SuppressLint("SimpleDateFormat")
    @BindingConversion
    @JvmStatic
    fun convertDateToDisplayTime(date: Date?): String {
        date?.let {
            return SimpleDateFormat("yyyy/MM/dd").format(it)
        }
        return ""
    }
}